user/caching.md

```diff
@@ -134,17 +134,27 @@ caches `$HOME/.cache/pip`.
 
 ### ccache cache
 
-For caching `ccache` files, use:
+If you are using `ccache`, use:
 
 ```yaml
 language: c # or other C/C++ variants
 
 cache: ccache
 ```
 
-caches `$HOME/.ccache`, and adds `/usr/lib/ccache` to the front of `$PATH`.
+to cache `$HOME/.ccache`, and add `/usr/lib/ccache` to the front of `$PATH`.
```

Comment:
In the case of Linux, Travis automatically edits the PATH when it detects that ccache is being used: https://github.com/travis-ci/travis-build/blob/35b5046996ef1f50d9be2206e921666e0937b8d5/lib/travis/build/script/c.rb#L12-L13